<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialogShortcuts</class>
 <widget class="QDialog" name="SettingsDialogShortcuts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Keyboard shortcuts</string>
  </property>
  <property name="windowIcon">
   <iconset theme="preferences-desktop-keyboard-shortcuts">
    <normaloff/>
   </iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <widget class="QTableWidget" name="shortcutsWidget">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string notr="true"/>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Action</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Key</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>Play / Pause</string>
      </property>
      <property name="flags">
       <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>Stop</string>
      </property>
      <property name="flags">
       <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>Next channel</string>
      </property>
      <property name="flags">
       <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string>Previous channel</string>
      </property>
      <property name="flags">
       <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="3" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="4" column="0">
      <property name="text">
       <string>Fullscreen</string>
      </property>
      <property name="flags">
       <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="4" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="5" column="0">
      <property name="text">
       <string>Toggle Info Widget</string>
      </property>
     </item>
     <item row="5" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="6" column="0">
      <property name="text">
       <string>Toggle Controls Widget</string>
      </property>
     </item>
     <item row="6" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="7" column="0">
      <property name="text">
       <string>Mute</string>
      </property>
      <property name="flags">
       <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="7" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="8" column="0">
      <property name="text">
       <string>Teletext</string>
      </property>
      <property name="flags">
       <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="8" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="9" column="0">
      <property name="text">
       <string>Volume up</string>
      </property>
      <property name="flags">
       <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="9" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="10" column="0">
      <property name="text">
       <string>Volume down</string>
      </property>
      <property name="flags">
       <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="10" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="11" column="0">
      <property name="text">
       <string>Recorder</string>
      </property>
      <property name="flags">
       <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="11" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="12" column="0">
      <property name="text">
       <string>Schedule</string>
      </property>
      <property name="flags">
       <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="12" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="13" column="0">
      <property name="text">
       <string>Current show info</string>
      </property>
     </item>
     <item row="13" column="1">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="14" column="0">
      <property name="text">
       <string>Open File</string>
      </property>
      <property name="flags">
       <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="14" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="15" column="0">
      <property name="text">
       <string>Open URL</string>
      </property>
      <property name="flags">
       <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="15" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="16" column="0">
      <property name="text">
       <string>Open Channel list</string>
      </property>
      <property name="flags">
       <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="16" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="17" column="0">
      <property name="text">
       <string>Settings</string>
      </property>
      <property name="flags">
       <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="17" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="18" column="0">
      <property name="text">
       <string>Keyboard shortcuts</string>
      </property>
     </item>
     <item row="18" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="19" column="0">
      <property name="text">
       <string>Always on top</string>
      </property>
      <property name="flags">
       <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="19" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="20" column="0">
      <property name="text">
       <string>Simple Mode</string>
      </property>
      <property name="flags">
       <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="20" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="21" column="0">
      <property name="text">
       <string>Hide to tray</string>
      </property>
      <property name="flags">
       <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="21" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="22" column="0">
      <property name="text">
       <string>Instant recording</string>
      </property>
     </item>
     <item row="22" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="23" column="0">
      <property name="text">
       <string>Take snapshot</string>
      </property>
     </item>
     <item row="23" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="24" column="0">
      <property name="text">
       <string>Preview channels</string>
      </property>
     </item>
     <item row="24" column="1">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="25" column="0">
      <property name="text">
       <string>Next audio track</string>
      </property>
     </item>
     <item row="25" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="26" column="0">
      <property name="text">
       <string>Next video track</string>
      </property>
     </item>
     <item row="26" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="27" column="0">
      <property name="text">
       <string>Next subtitle track</string>
      </property>
     </item>
     <item row="27" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="28" column="0">
      <property name="text">
       <string>Next aspect ratio mode</string>
      </property>
     </item>
     <item row="28" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="29" column="0">
      <property name="text">
       <string>Next crop mode</string>
      </property>
     </item>
     <item row="29" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
     <item row="30" column="0">
      <property name="text">
       <string>Next scale/zoom mode</string>
      </property>
     </item>
     <item row="30" column="1">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="31" column="0">
      <property name="text">
       <string>Next deinterlace mode</string>
      </property>
     </item>
     <item row="31" column="1">
      <property name="text">
       <string notr="true">-</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QtKeySequenceEdit" name="keyEditor" native="true">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="buttonClear">
     <property name="maximumSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
     <property name="icon">
      <iconset theme="edit-clear">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="buttonSet">
     <property name="text">
      <string>Set</string>
     </property>
     <property name="icon">
      <iconset theme="dialog-ok">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="shortcutsSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="buttonDefaults">
     <property name="text">
      <string>Defaults</string>
     </property>
     <property name="icon">
      <iconset theme="view-refresh">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtKeySequenceEdit</class>
   <extends>QWidget</extends>
   <header>common/QtKeySequenceEdit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
